### PR TITLE
Clean up JdbcSqlListExecutionRepository.

### PR DESCRIPTION
**Pull request by agorina**
There were two implementations that seemed almost identical.
And I will most likely need another one for the aggregate reports migrate. So I have decided to clean it up.

### CODE REVIEW

File: common/src/main/java/org/opentestsystem/rdw/ingest/common/repository/JdbcSqlListExecutionRepository.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ * JDBC implementation of a SqlListExecutionRepository.
+ */
+public class JdbcSqlListExecutionRepository implements SqlListExecutionRepository {
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+
+    public JdbcSqlListExecutionRepository(final NamedParameterJdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    @Transactional
```
Comment (agorina) [inline common/src/main/java/org/opentestsystem/rdw/ingest/common/repository/JdbcSqlListExecutionRepository.java]: Group's implementation did not have this annotation. @wtritch, do you have any concerns about this change?

File: group-processor/src/main/java/org/opentestsystem/rdw/ingest/group/repository/JdbcSqlListExecutionRepository.java [hunk 2]
```diff
@@ -1,37 +1,17 @@
...
 import org.springframework.stereotype.Repository;
 
-import java.util.Map;
-
 /**
  * JDBC implementation of a SqlListExecutionRepository.
  */
 @Repository
-public class JdbcSqlListExecutionRepository implements SqlListExecutionRepository {
+public class JdbcSqlListExecutionRepository extends org.opentestsystem.rdw.ingest.common.repository.JdbcSqlListExecutionRepository {
```
Comment (agorina) [inline group-processor/src/main/java/org/opentestsystem/rdw/ingest/group/repository/JdbcSqlListExecutionRepository.java]: Better name?

Comment (malaffoon) [reply group-processor/src/main/java/org/opentestsystem/rdw/ingest/group/repository/JdbcSqlListExecutionRepository.java]: `DefaultSqlListExecutionRepository`?

File: migrate-reporting/src/main/java/org/opentestsystem/rdw/ingest/migrate/reporting/repository/impl/JdbcReportingSqlListExecutionRepository.java [hunk 3]
```diff
@@ -1,42 +1,18 @@
...
-            jdbcTemplate
-                    .getJdbcOperations()
-                    .execute(sql);
-        }
-    }
-
-    @Transactional
-    @Override
-    public void execute(final Iterable<String> sqls, final Map<String, ?> parameters) {
+class JdbcReportingSqlListExecutionRepository extends JdbcSqlListExecutionRepository {
```
Comment (malaffoon) [inline migrate-reporting/src/main/java/org/opentestsystem/rdw/ingest/migrate/reporting/repository/impl/JdbcReportingSqlListExecutionRepository.java]: `ReportingSqlListExecutionRepository`?

File: common/src/main/java/org/opentestsystem/rdw/ingest/common/repository/JdbcSqlListExecutionRepository.java (hunk 1 above)
Comment (wtritch) [reply common/src/main/java/org/opentestsystem/rdw/ingest/common/repository/JdbcSqlListExecutionRepository.java]: Nope, I believe the group annotations are at the service layer. Having two transaction annotations shouldn't change anything.

**Pull request merged**